web/src/main/java/com/microsoft/applicationinsights/web/internal/WebRequestTrackingFilter.java

```diff
@@ -59,20 +58,24 @@
     private boolean agentIsUp = false;
     private final LinkedList<ThreadLocalCleaner> cleaners = new LinkedList<ThreadLocalCleaner>();
 
+    //Only for Spring Boot Application (External Name Setting)
```

Comment:
@dhaval24 , Thank for looking into this! Then the question is - will changing name right in the code affect the behavior of the SDK in some unpredictable way?
For example, if name from the code takes precedence, could it be that some configuration settings are not picked up (simply because they are mapped to the name used in the configuration but not in code)? If no such or similar case is possible - we should be OK with this change.